### DISCUSSION TITLE
关于文章列表配图&内容宽度等修改，wordpress字符转义相册显示异常等问题

### DISCUSSION THREAD

**Discussion by GowanOAtta** [mirai-mamori/Sakurairo#362] [Q&A]
请问
1、主页文章列表配图与摘要的宽度占比怎么改？
![图片](https://user-images.githubusercontent.com/24862657/139683288-d2e0980f-756a-40bf-9f28-bcbfdcb37de3.png)

2、我加宽 style.css 中 .site-content 的宽度，在非主页的那个文章列表元素间距也变了
![图片](https://user-images.githubusercontent.com/24862657/139679216-86ea8405-164b-4c1e-8c3d-c89a60584ac5.png)
请问
1) 文章摘要前面的小图怎么换？
2) 文章摘要与小图之间的距离怎么改？
3) 鼠标悬停时显示的评论、热度等信息的位置怎么改？
4) 鼠标悬停在小图上显示的黑底图片怎么去除？
![图片](https://user-images.githubusercontent.com/24862657/139680868-7c2642d6-40f1-4803-b9d3-7098cd7d6431.png)

3、加宽内容区域的宽度后，进入文章，章节目录与内容有遮挡，可以改为目录+内容的宽度为 .site-content 的宽度吗？
![图片](https://user-images.githubusercontent.com/24862657/139670696-5a7b5bdc-3cce-46a1-8c02-9650a312d8ba.png)

4、相册显示“未上传图片或图片设置异常”
代码按如下添加了
![图片](https://user-images.githubusercontent.com/24862657/139573643-4554f0fe-d5a9-4411-8011-8e3944ad08d6.png)
`&lt;`是wordpress自己改的，在代码编辑模式下也这样，我改回好几次都不行，不点在编辑框内就这样了，还以为是wordpress的特性。我查了说需要在 functions.php 中加 remove_filter('the_content', 'wptexturize'); 取消转义，可是这个文件中原来就有了。我又在相册模板中加也不行。
![图片](https://user-images.githubusercontent.com/24862657/139573622-4ad2d29c-a069-4dcb-80e6-3244951f4491.png)
@KotoriK 在#361 提到把板块改成自定义html，是指自定义模板吗，有没有样本？

**Comment by KotoriK** [comment]
回答4.
古腾堡有一个自定义HTML板块可以查看
另外html标签中 标签名不用和括号有空格 即不应该是< img>而是\<img>

**Reply by GowanOAtta** [reply]
@KotoriK 
> 
> 
> 回答4. 古腾堡有一个自定义HTML板块可以查看 另外html标签中 标签名不用和括号有空格 即不应该是< img>而是<img>

OK了，我用了 markdown 的插件就不转义了，再把 < img> 中的空格去掉，用原来的相册模板就成功了
